packages/mapper/src/mapper.ts

```diff
@@ -81,9 +89,12 @@ export default class Mapper {
       case 'dbs':
         const result = await this.serviceProvider.listDatabases('admin');
         if (!('databases' in result)) {
-          throw new Error('Error: invalid result from listDatabases');
+          const err = new Error('Error: invalid result from listDatabases');
+          this.messageBus.emit('error', err);
```

Comment:
Would it be an option to handle the error logging only once? Dunno, maybe with a `Proxy`, so we avoid to test that we emit the error each time there is an exception.

Reply:
Yea, I think that'd be possible. 

I already do this in the CLI since that's where we catch all the errors, so I think this might be already redundant call. Good catch!